app/src/organisms/ODD/QuickTransferFlow/SelectTipFrequency.tsx

```diff
@@ -38,7 +38,7 @@ export function SelectTipFrequency({
     setSelectedChangeTipOption,
   ] = useState<ChangeTipOptions>()
 
-  const allowedChangeTipOptions: ChangeTipOptions[] = ['once']
+  const allowedChangeTipOptions: ChangeTipOptions[] = ['once', 'never']
```

Comment:
we can't actually achieve a tip frequency of never since there is only ever 1 transfer, consolidate or distribute step generated with QT

```suggestion
  const allowedChangeTipOptions: ChangeTipOptions[] = ['once']
```

Reply:
thank you.